StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSavedPaymentMethodsCollectionViewController.swift

```diff
@@ -498,7 +498,15 @@ extension CustomerSavedPaymentMethodsCollectionViewController: PaymentOptionCell
 extension CustomerSavedPaymentMethodsCollectionViewController: UpdatePaymentMethodViewControllerDelegate {
     func didUpdate(viewController: UpdatePaymentMethodViewController,
                    paymentMethod: STPPaymentMethod,
-                   updateParams: StripePayments.STPPaymentMethodUpdateParams) async throws {
+                   updateParams: UpdatePaymentMethodParams) async throws {
+        guard let updateCardBrandParams = updateParams.updateCardBrandParams else {
```

Comment:
With the recent refactor, we can remove this type`UpdatePaymentMethodParams`, and just use:
```
        if let updateParams = viewController.updateParams,
           case .card(let paymentMethodCardParams) = updateParams {
        }
```
Similarly, UpdatePaymentViewController could have:
```
    var setAsDefaultValue: Bool? {
        guard hasChangedDefaultPaymentMethodCheckbox,
              let checkbox = setAsDefaultCheckbox else {
            return nil
        }
        return checkbox.isSelected
    }
```
so that callers can:
```
    if let setAsDefault = viewController.setAsDefaultValue {
    }
```